low-level/sensors/impl/centos_7/drive_manager.py

```diff
@@ -59,6 +59,12 @@ def name():
         """@return: name of the monitoring module."""
         return DriveManager.SENSOR_NAME
 
+    @staticmethod
+    def impact():
+        """Returns impact of the module."""
+        return ("Drive path changes can not be monitored "

```

Comment:
Drive path changes? Write from end consumer perspective, think layman is reading this impact string

Reply:
Done. Please take a look.